### DISCUSSION TITLE
Streamline process of entering and maintaining anchors

### DISCUSSION THREAD

**Discussion by beccon4** [freeplane/freeplane#1554] [Ideas]
Would it be possible to rearrange the context menu to make it quicker to set, maintain and link to anchors?

For the time beeing it is in the 3rd level <right click> -> Insert (*) -> Insert Link -> (and then quite below) -> {set | link from | link to | delete} Anchor which is quite awkward.  (*) to make matters worse there is another entry "Insert" in the context menu which refers to paste (in German both "einfügen")

Well even the secound menu level should be in the first one <right click>-> insert node | insert image ;-) (should work with paste) | insert link |  insert anchor | insert the more exotic things as menu items or system commands.

**Comment by Illegit** [comment]
In the meantime I suggest using a combination of shortcuts: in my case I use 
1) when selecting the anchor node --> ctrl D for setting the anchor 
2) when selecting the to node --> alt D for anchor to and alt+shift D for anchor from

Just if it can help.
Regards
